ibm_catalog.json

```diff
@@ -0,0 +1,250 @@
+                  "caption": "Observability agents on a Red Hat OpenShift cluster",
+                  "type": "image/svg+xml"
+                },
+                "description": "Observability agents deployed on a Red Hat OpenShift cluster."
+              }
+            ]
+          },
+          "dependencies": [
+            {
+              "flavors": [
+                "quickstart",
+                "standard"
+              ],
+              "id": "95fccffc-ae3b-42df-b6d9-80be5914d852-global",
+              "name": "deploy-arch-ibm-slz-ocp",
+              "version": ">=v3.0.0",
+              "optional": true
+            },
+            {
+              "flavors": [
+                "instances"
+              ],
+              "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+              "id": "a3137d28-79e0-479d-8a24-758ebd5a0eab-global",
+              "name": "deploy-arch-ibm-observability",
+              "version": ">=v1.3.1",
+              "optional": true
+            }
+          ],
+          "dependency_version_2": true,
+          "configuration": [
+            {
+              "key": "ibmcloud_api_key",
+              "required": true
+            },
+            {
+              "key": "prefix",
+              "required": true,
+              "description": "The prefix to add to all resources that this solution creates. To not use any prefix value, you can enter the string `__NULL__`."
+            },
+            {
+              "key": "cluster_id",
+              "type": "string",
+              "default_value": "",
+              "display_name": "Cluster",
+              "required": true,
+              "custom_config": {
+                  "type": "cluster_var",
+                  "grouping": "deployment",
+                  "original_grouping": "deployment"
+              }
+            },
+            {
+              "key": "cluster_resource_group_id",
+              "type": "string",
+              "default_value": "",
+              "display_name": "Cluster resource group",
+              "required": true,
+              "custom_config": {
+                  "type": "resource_group",
+                  "grouping": "deployment",
+                  "original_grouping": "deployment",
+                  "config_constraints": {
+                      "identifier": "rg_id"
+                  }
+              }
+            },
+            {
+              "key": "is_vpc_cluster",
+              "required": true
+            },
+            {
+              "key": "cloud_monitoring_enabled"
+            },
+            {
+              "key": "cloud_monitoring_access_key"
+            },
+            {
+              "key": "cloud_monitoring_add_cluster_name"
+            },
+            {
+              "key": "cloud_monitoring_agent_name"
+            },
+            {
+              "key": "cloud_monitoring_agent_namespace"
+            },
+            {
+              "key": "cloud_monitoring_agent_tags"
```

Comment:
tags should use custom config like this:
```
{
    "key": "cloud_monitoring_agent_tags",
    "custom_config": {
      "grouping": "deployment",
      "original_grouping": "deployment",
      "config_constraints": {
        "type": "string"
      }
  }
```
Please review the guidance in https://github.ibm.com/GoldenEye/issues/issues/12542